docs/2019-08-05-new-design-of-backup-restore.md

```diff
@@ -0,0 +1,233 @@
+Last updated: 2020-03-07
```

Comment:
I guess it could be better to rename the directory to `docs-cn`.

Reply:
How about `docs/cn`?